guides/cheat-sheets/voice.cheatmd

```diff
@@ -0,0 +1,173 @@
+# Voice Usage
+
+This cheat sheet covers basic use of the Discord Voice API through the `Nostrum.Voice` module.
+
+## Playing Audio
+{: .col-2}
+
+### Playing immediately with a `try_play` function
+
+```elixir
+def try_play(guild_id, url, type, opts \\ []) do
+  case Nostrum.Voice.play(guild_id, url, type, opts) do
+    {:error, _msg} ->
+      # Wait for handshaking to complete
+      Process.sleep(100)
+      try_play(guild_id, url, type, opts)
+
+    _ ->
+      :ok
+  end
+end
+```
+
+```elixir
+Nostrum.Voice.join_channel(guild_id, channel_id)
+try_play(guild_id, "./song.mp3", :url)
+```
+
+### Playing immediately using `Nostrum.Voice.ready?/1`
+
+```elixir
+def play_when_ready(guild_id, url, type, opts \\ []) do
+  if Nostrum.Voice.ready?(guild_id) do
+    Nostrum.Voice.play(guild_id, url, type, opts)
+  else
+    # Wait for handshaking to complete
+    Process.sleep(25)
+    play_when_ready(guild_id, url, type, opts)
+  end
+end
+```
+
+```elixir
+Nostrum.Voice.join_channel(guild_id, channel_id)
+play_when_ready(guild_id, "https://youtu.be/b4RJ-QGOtw4", :ytdl)
+```
+
+### Playing immediately by using events
+
+```elixir
+alias Nostrum.Struct.Event.VoiceReady
+alias Nostrum.Voice
+```
+
+In your consumer module
+
+```elixir
+def handle_event({:VOICE_READY, %VoiceReady{guild_id: guild_id} = _event, _v_ws_state}) do
+  Voice.play(guild_id, "~/loud_noise.mp3", :url, volume: 10)
+end
+```
```

Comment:
I would bundle these together for the sake of brevity in the cheatsheet, I can't use suggestions because codeblock formatting gets weird, but I'd remove the "In your consumer module bit" in favour of just a codeblock that has:

```elixir
alias Nostrum.Struct.Event.VoiceReady
alias Nostrum.Voice

def handle_event({:VOICE_READY, %VoiceReady{guild_id: guild_id} = _event, _v_ws_state}) do
  Voice.play(guild_id, "~/loud_noise.mp3", :url, volume: 10)
end
```